### ISSUE TITLE
Stream argument 'disabled' don't change stream status

### ISSUE THREAD

**Issue by saidogoid** [suzuki-shunsuke/go-graylog#232]
## Environment

* Terraform version:  0.11.14
* Graylog version: v3.1.4+1149fe1 Docker
* terraform-provider-graylog: graylog 10.0.1
* OS (client side): Debian 10

## Overview

When you specify `disabled = true` it shoud disabled the stream, and the opposite. Not working here either of options.

## Expected behavior

Expected to obey of terraform resource to change the status of stream.

## Actual behavior

No changing behavior, even if changed when running apply.

`graylog_stream.stream_test: Modifying... (ID: 5e2189ae491a49001210bb78)
  disabled: "false" => "true"`
## How to reproduce

```
resource "graylog_index_set" "test" {
  title                               = "example"
  description                         = "Managed by Terraform"
  index_prefix                        = "prefix"
  rotation_strategy_class             = "org.graylog2.indexer.rotation.strategies.MessageCountRotationStrategy"
  retention_strategy_class            = "org.graylog2.indexer.retention.strategies.DeletionRetentionStrategy"
  description                         = "test 2"
  index_analyzer                      = "standard"
  index_optimization_disabled         = true
  writable                            = true
  shards                              = 4
  replicas                            = 0
  index_optimization_max_num_segments = 1
  field_type_refresh_interval         = 5000

  retention_strategy {
    max_number_of_indices = 5
    type                  = "org.graylog2.indexer.retention.strategies.DeletionRetentionStrategyConfig"
  }

  rotation_strategy {
    max_docs_per_index = 20000
    max_size           = 0
    type               = "org.graylog2.indexer.rotation.strategies.MessageCountRotationStrategyConfig"
  }
}

resource "graylog_stream" "stream_test" {
  title                              = "test"
  description                        = "Managed by Terraform"
  index_set_id                       = "${graylog_index_set.test.id}"
  disabled                           = true
  matching_type                      = "AND"
  remove_matches_from_default_stream = true
}
```
Any help on this?
Am I doing something wrong?

**Comment by suzuki-shunsuke** [comment]
Thank you for your feedback.
I'll check.

**Comment by suzuki-shunsuke** [comment]
https://github.com/suzuki-shunsuke/go-graylog/pull/234

**Comment by suzuki-shunsuke** [comment]
I could reproduce the bug and fixed it by #234.
I have released [v1.0.2-1](https://github.com/suzuki-shunsuke/go-graylog/releases/tag/v10.0.2-1).
Please check.
If there is no problem, I'll release v1.0.2 later.

**Comment by saidogoid** [comment]
> 
> 
> I could reproduce the bug and fixed it by #234.
> I have released [v1.0.2-1](https://github.com/suzuki-shunsuke/go-graylog/releases/tag/v10.0.2-1).
> Please check.
> If there is no problem, I'll release v1.0.2 later.

Thx! Solved on the new version that you deployed!
Thx for the fastest response and bug solve.

Great Job!
